02week/bio/index.html

```diff
@@ -0,0 +1,52 @@
+      the injury to his elbow resulted in a set back of all the work he had put in to rehabilitating his thumb. Because of these two injuries, Nick was unable to play guitar for more than 18 months. But with great determination and a passionate focus,
+      he managed to regain his ability to play his beloved guitar. Truth is, playing the guitar again was never in doubt; giving up his dream was not an option. In fact, with great perserverance, Nick was able to take his craft to another level.</p>
+
+    <p>In 2002, Nick attended the University of Texas Pan-American. There he enrolled in the Music of Performing Arts bachelor degree program. During his studies at UTPA, Nick received classical training in guitar performance. While at UTPA, he played on
+      many guitar ensembles representing the universitys music department. He also had the opportunity to help tutor some of his fellow students enrolled in the degree program. In 2006, Nick decided to take his musical education to Hollywood, California
+      where he attended the world famous Musicians Institute. There he enrolled in the Guitar Institute of Technology Associates program, where Nick learned the arts of Fusion, Blues, and Jazz. While in Hollywood, Nick played with many different up-and-coming
+      artists with styles ranging from Country/Rock, Blues, and Jazz/Fusion music. In 2008, Nick graduated with honors from Musicians Institute and is currently pursuing other musical opportunities.</p>
+
+    <p>Nick has both shared the stage with, and learned from, several great artists, including: Jake Eckert (Dirty Dozen Band, New Soul Underground), Stewart Ziff (War), Ken Stieger (Project Stieger), Alex Machacek, Antti Kotikoski, Allen Hinds (Artist/LA
+      Session player/Gino Vinelli), Dan Gilbert (MI Director of Programs/Instructor), Bruce Buckingham (Michael Jackson), Dale Turner (Writer for Guitar Player Magazine), and Maurice Verloop.</p>
+
+    <p>Nick Andrews now resides in Austin, Texas where he does sessions for local artists and tours with Texas Country Acts. Nick is currently playing with artist Bart Crow, promoting his sixth album "The Parade". Nick has also played with the following,
+      Bill Rice (Souncheck International Mgt) as the showcased lead guitarist promoting the album "Long Hard Ride" with Aaron Ave Records. Nick has also toured as the lead guitarists for recording artist Dean Seltzer (Smith Entertainment). While playing
+      the Texas Country Circuit Nick has openened for Eric Church on the Jagermeister Country Tour and has also opened up for the following artists: Jack Ingram, Joe Nicols, Eli Young Band, Cory Morrow, Wade Bowen, Stoney LaRue, Jason Boland, Point Blank,
+      Bad Company, Sammy Kershaw, Mark Chestnut, Brandon Jenkins, Aaron Watson, Kyle Park, Roger Creager.</p>
+  </div>
+
+
+  <div class="footer">
```

Comment:
I believe we covered HTML semantic elements, and one of them was actually `footer`.